mtg/packs.go

```diff
@@ -0,0 +1,81 @@
+package mtg
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+type Pack struct {
+	Cards []*Card
+}
```

Comment:
We might want to have a `NewPack` function that creates a new pack and builds the underlying array;   I think it ends up being `nil` until you explicitly create it.

```
func NewPack() *Pack {
    return &Pack{
        Cards: []*Card{},
    }
}
```